cms/envs/aws.py

```diff
@@ -87,6 +87,20 @@
     DEFAULT_PRIORITY_QUEUE: {}
 }
 
+# Setup alternate queues, to allow access to cross-process workers
+ALTERNATE_QUEUE_ENVS = os.environ.get('ALTERNATE_SERVICE_VARIANTS', '').split()
```

Comment:
I think we should probably just name this ALTERNATE_CELERY_QUEUES or ALTERNATE_WORKER_QUEUES (naming is hard, I'm not sure how folks usually talk about these queues) maybe ADDITIONAL_(CELERY|WORKER)_QUEUES but I think I'm ok with this approach.